src/android/activity/UsabillaActivity.java

```diff
@@ -83,24 +83,13 @@ private void setUpBroadcastReceivers() {
         receiverFormClosed = new BroadcastReceiver() {
             @Override
             public void onReceive(Context context, Intent intent) {
-                getSupportFragmentManager().executePendingTransactions();
-
-                List<Fragment> fragments = getSupportFragmentManager().getFragments();
-                DialogFragment currentFragment = (DialogFragment) fragments.get(fragments.size() - 1);
-                Dialog ratingDialog = currentFragment.getDialog();
-
-                if (ratingDialog == null) {
-                    finishActivity();
-                    return;
-                }
-
-                ratingDialog.setOnDismissListener(dialog -> finishActivity());
+                finishActivity(intent);
```

Comment:
why has this been removed? has something changed in the handling of the PlayStore dialog?

Reply:
I think this is removed because of the bug which occurs during the dismiss that from play store dialog its not going back to the activity